xcell/tests/test_mapper_p18cmbk.py

```diff
@@ -30,9 +41,16 @@ def test_get_signal_map():
 
 
 def test_get_mask():
-    m = get_mapper()
+    c = get_config()
+    c['path_lite'] = './lite'
```

Comment:
I like the idea of using a tmp directory for the lite maps/catalogs, instead of the `data` one.